core/src/main/java/org/frankframework/jdbc/datasource/FrankResources.java

```diff
@@ -79,7 +79,7 @@ public String toString() {
 		Class<?> clazz = ClassUtils.loadClass(resource.getType());
 
 		if(lookupClass.isAssignableFrom(DataSource.class) && Driver.class.isAssignableFrom(clazz)) { //It's also possible to use the native drivers instead of the DataSources directy.
```

Comment:
Wat ik me opeens bedenk is: willen we dit ook voor XADataSources kunnen doen? Een XADataSource maken wanneer de Driver class geconfigureerd is ipv een XADataSource class?

(Buiten scope van dit PR maar ik wou het toch even aanstippen)

Reply:
Tja, hangt van de door de gebruiker geconfigureerde driver af.